netbout-web/src/main/java/com/netbout/rest/bout/FkBout.java

```diff
@@ -57,60 +55,54 @@
      * @param take Take
      */
     FkBout(final String regex, final Take take) {
-        super(
-            new Fork() {
-                @Override
-                public Opt<Response> route(final Request req)
-                    throws IOException {
-                    return FkBout.route(regex, take, req);
-                }
-            }
-        );
+        this(regex, take, true);
     }
 
     /**
-     * Route.
+     * Ctor.
      * @param regex Regular expression
      * @param take Take
-     * @param req Request
-     * @return Response or empty
-     * @throws IOException If fails
+     * @param redirect Redirect to bout page flag
      */
-    private static Opt<Response> route(final String regex, final Take take,
-        final Request req) throws IOException {
-        return new FkRegex(
-            String.format("/b/([0-9]+)%s", regex),
-            new TkRegex() {
-                @Override
-                public Response act(final RqRegex rreq) throws IOException {
-                    final long bout = Long.parseLong(
-                        rreq.matcher().group(1)
-                    );
-                    return FkBout.redirect(bout, take).act(
-                        new RqWithHeader(
-                            rreq, "X-Netbout-Bout",
-                            Long.toString(bout)
-                        )
-                    );
+    FkBout(final String regex, final Take take, final boolean redirect) {
+        super(
+            new FkRegex(
+                String.format("/b/([0-9]+)%s", regex),
```

Comment:
@exper0 OK, I see now, you moved it, no need for it,  this would be a mire general issue